test/e2e/common/pod_resize.go

```diff
@@ -241,10 +265,24 @@ func verifyPodContainersCgroupConfig(pod *v1.Pod, tcInfo []TestContainerInfo) {
 			} else {
 				cpuShares = int64(kubecm.MilliCPUToShares(cpuRequest.MilliValue()))
 			}
+
 			cpuQuota := kubecm.MilliCPUToQuota(cpuLimit.MilliValue(), kubecm.QuotaPeriod)
-			verifyCgroupValue(ci.Name, CgroupCPUShares, strconv.FormatInt(cpuShares, 10))
-			verifyCgroupValue(ci.Name, CgroupCPUQuota, strconv.FormatInt(cpuQuota, 10))
-			verifyCgroupValue(ci.Name, CgroupMemLimit, strconv.FormatInt(memLimitInBytes, 10))
+
+			// Added by chenw. Verify the CPU shares only when the pod QoS is guaranteed.
+			//if (pod.Status.QOSClass == v1.PodQOSGuaranteed) {
```

Comment:
no commented out code please

Reply:
Yes, I should remove this line. It was used to debug the code.